corehq/warehouse/models/staging.py

```diff
@@ -40,21 +45,15 @@ class GroupStagingTable(StagingTable):
 
     group_last_modified = models.DateTimeField(null=True)
 
-    @classmethod
-    @transaction.atomic
-    def stage_records(cls, start_dateime, end_datetime):
-        cls.clear_records()
-
-        records = []
-        for raw_record in cls.raw_record_iter(start_dateime, end_datetime):
-            records.append(GroupStagingTable(
-                group_id=raw_record.get('_id'),
-                name=raw_record.get('name'),
-                case_sharing=raw_record.get('case_sharing'),
-                reporting=raw_record.get('reporting'),
-                group_last_modified=raw_record.get('last_modified'),
-            ))
-        cls.objects.bulk_create(records)
+    # Map source model fields to staging table fields
+    # ( <source field>, <staging field> )
+    FIELD_MAPPING = [
```

Comment:
it might be a good idea to make this a property or be returned from a classmethod, so that we can put it on the superclass and raise a NotImplementError, just so it's clear what needs to be implemented on a new staging table model